code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -232,18 +234,36 @@
 	if(!length(querys))
 		return
 	var/list/datum/SDQL2_query/running = list()
+	var/list/datum/SDQL2_query/waiting_queue = list() //Sequential queries queue.
+
 	for(var/list/query_tree in querys)
 		var/datum/SDQL2_query/query = new /datum/SDQL2_query(query_tree)
 		if(QDELETED(query))
 			continue
 		if(usr)
 			query.show_next_to_key = usr.ckey
+		waiting_queue += query
+		//This needs to be done before Run() is used, unlike every other query option, so we're checking it here and we're doing it differently
+		if(query.options && findtext(query_text, "sequential = true"))
+			sequential = TRUE
```

Comment:
The original TG PR set a define and then checked a value on it like how the other options do it. However, that doesn't work, even on TG, and I believe the reason for that is because the defines are largely meaningless until set_option() is called, which comes after Run() - which we need to go ***before***.

This is definitely a pretty shitty way to go about doing this, but it works, and I can't come up with anything better.
I will gladly accept any alternative ways to go about this.